server/block/sea_pickle.go

```diff
@@ -0,0 +1,165 @@
+package block
+
+import (
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/item"
+	"github.com/df-mc/dragonfly/server/world"
+	"github.com/go-gl/mathgl/mgl64"
+	"math"
+	"math/rand"
+)
+
+// SeaPickle is a small stationary underwater block that emits light, and is typically found in colonies of up to
+// four sea pickles.
+type SeaPickle struct {
+	empty
+	transparent
+
+	// ClusterCount is the amount of additional sea pickles clustered together.
+	ClusterCount int
```

Comment:
Would maybe just name this `Count`.

Reply:
That said, it seems this is 0 when there's 1 pickle. We should either document that very well or change the name of this field in a way that it indicates additional pickles.